refiner-lib/src/near_stream.rs

```diff
@@ -109,11 +109,23 @@ impl NearStream {
 
         blocks
     }
+
+    pub async fn next_explicit_block(&mut self, near_block: &NEARBlock) -> Vec<AuroraBlock> {
```

Comment:
From the perspective of future maintenance on this codebase, I worry about having a second function which duplicates most of the code from `next_block`, except it doesn't include skip blocks. For now it is only used in tests, but in tests I think we can always ensure there are no skip blocks any way (e.g. by directly mutating the height field of the stream), so only having `next_block` should be sufficient.

Reply:
fixed